src/parser/jobs/ast/modules/CastTime.tsx

```diff
@@ -0,0 +1,28 @@
+import {Event} from 'event'
+import {filter} from 'parser/core/filter'
+import CoreCastTime from 'parser/core/modules/CastTime'
+
+export class CastTime extends CoreCastTime {
+	private pomIndex: number | null = null
+
+	override initialise() {
+		super.initialise()
+
+		const presenceOfMindFilter = filter<Event>()
```

Comment:
var naming nitpick, can we modify this filter from WHM to reflect HoM by name?

Reply:
lol not nitpicky at all
I should use my eyes when copying instead of committing and praying lol
it's been updated